tests/main/views/test_marketplace.py

```diff
@@ -451,7 +451,7 @@ def test_dos_brief_has_application_stats_correctly_when_no_applications(self):
 
         assert started_responses_section.xpath('div[@class="big-statistic"]/text()')[0] == '0'
         assert completed_responses_section.xpath('div[@class="big-statistic"]/text()')[0] == '0'
-        assert started_responses_section.xpath('div[@class="statistic-name"]/text()')[0] == "Started applications"
+        assert started_responses_section.xpath('div[@class="statistic-name"]/text()')[0] == "Incomplete applications"
```

Comment:
you mean serialising those assertions? I was thinking about it, in general I follow the rule that if similar code appears 1-2 times, I leave it as is, if it's >=3, then I serialise / refactor for DRYness, but I'm open to change my ways.